install.sh

```diff
@@ -138,15 +138,6 @@ Install()
 
```

Comment:
We can update the Copyright for this file too (Github doesn't allow a comment in that line)

Reply:
Fixed